src/services/getip.ts

```diff
@@ -4,36 +4,40 @@ import { logger } from "./logger"
 
 export const getIP = (
   headers: Headers,
-  checkHeaders: IPHeaders[] = headersToCheck,
+  checkHeaders: IPHeaders[] | IPHeaders = headersToCheck,
```

Comment:
Why can't we just let it remain an Array? What benefit does a string option provide us? Imo we will be slowing down the application this way

Reply:
I see on old code, It already support string checkHeaders.
I just add type and improve getIP.
https://github.com/gaurishhs/elysia-ip/blob/bd53e7fc8bb573a91cc3bfa33795d95174c97790/src/services/getip.ts#L9-L23

Reply:
I get that but it was added by you in prev. PR this seems to be just increasing the complexity

Reply:
https://github.com/gaurishhs/elysia-ip/blob/f46952e0baa39e63aaabe19a2ada291e8ee87036/src/index.ts#L19-L30